test/controllers/api/tracks/trophies_controller_test.rb

```diff
@@ -13,8 +13,9 @@ class API::Tracks::TrophiesControllerTest < API::BaseTestCase
     get api_track_trophies_url(track.slug), headers: @headers, as: :json
 
     assert_response :ok
-    expected = SerializeTrackTrophies.(track, User.first).to_json
-    assert_equal expected, response.body
+    expected = { trophies: SerializeTrackTrophies.(track, User.first) }
+    actual = JSON.parse(response.body, symbolize_names: true)
+    assert_equal expected, actual
```

Comment:
I think you probably want to keep the structure that was there before (to avoid JSON serialization comparison issues)

```suggestion
    expected = { trophies: SerializeTrackTrophies.(track, User.first) }.to_json
    assert_equal expected, response.body
```